Development/nmos/sdp_utils.cpp

```diff
@@ -192,6 +192,14 @@ namespace nmos
         return{ sender.at(nmos::fields::label).as_string(), params, 100, media_stream_ids, details::make_ts_refclk(node, source, sender) };
     }
 
+    static sdp_parameters make_mux_sdp_parameters(const web::json::value& node, const web::json::value& source, const web::json::value& flow, const web::json::value& sender, const std::vector<utility::string_t>& media_stream_ids)
+    {
+        sdp_parameters::mux_t params;
+        params.tp = sdp::type_parameters::type_NL;
```

Comment:
Why default to Narrow Linear here?

Reply:
ST2022-8 Section 6 specifies that

> Senders shall comply with either the "Narrow Linear Senders (Type NL)" requirements, or the "Wide Senders (Type W)" requirements.

Reply:
Ah! How did I miss that? More haste, less speed reading. Sorry! Makes sense. Only Gapped is irrelevant.